runtime/README.md

```diff
@@ -281,6 +281,35 @@ Command Code: `0x5349_4756` ("SIGV")
 | chksum      | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips\_status | u32      | Indicates if the command is FIPS approved or an error.
 
+### LMS\_SIGNATURE\_VERIFY
```

Comment:
Should note somewhere here that if signature verification fails it will be returned as an fw_non_fatal error code (rather than in the response).

Reply:
Good idea, will do